### PR TITLE
LG5 unit test

### PR DESCRIPTION
**Pull request by Meowitsmadi**
# Description

- Implemented test LG5: "should barely have hourly points for middle readings of 15 + 20 minute for a 15 day + 15 min period and quantity units with kWh as kWh".
- Added the 9-21 to 10-06 csv file

Co-authored by: @DawnSovereign - Jason Kim, jason.t.kim2@gmail.com
Co-authored by: @Pimi831 - Julian Ochoa Villicana, julian.vochoa@gmail.com

Fixes #962  

## Type of change

- [ ] Note merging this changes the database configuration.
- [x] This change requires a documentation update

## Checklist

- [x] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [x] I have removed text in ( ) from the issue request
- [x] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

### CODE REVIEW

**Comment by huss** [comment]
Thanks to @Meowitsmadi for this pull request. I don't see a CLA from @DawnSovereign or @Pimi831. Could they please use the link in the Description/Checklist above to compete it. If you think my records are wrong on the CLA then please let me know.

**Comment by Meowitsmadi** [comment]
Hi @huss, I've let them know, thanks.

**Comment by Meowitsmadi** [comment]
@huss The CLAs should be signed now. Please let me know if you need any additional information, thanks!

File: package-lock.json [hunk 1]
```diff
@@ -2784,8 +2784,7 @@
 		"node_modules/@types/lodash": {
 			"version": "4.14.201",
 			"resolved": "https://registry.npmjs.org/@types/lodash/-/lodash-4.14.201.tgz",
-			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ==",
-			"dev": true
+			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ=="
```
Comment (huss) [inline package-lock.json]: Could the ``"dev": true`` line be put back in. Even if it is an appropriate change (would need to check), I don't think it should be part of this PR.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @Meowitsmadi, @DawnSovereign & @Pimi831 for their first pull request. The test runs as expected. I made one small comment about removing a small change in another file. Once that is addressed this should be ready to merge.

File: package-lock.json (hunk 1 above)
Comment (Meowitsmadi) [reply package-lock.json]: Hello, I tried to pull before pushing the change but it seems I have accidentally tried to merge my forked development branch with the LG5 branch (?) and I can't push the change as it keeps asking me to merge. Is there a place to message you further to resolve this? I'm a new developer/contributor and I don't want to mess anything up or move forward without understanding what I'm doing. Thanks

Comment (Meowitsmadi) [reply package-lock.json]: Disregard my last message, the dev: true should be back now, sorry for the confusion!

**Comment by huss** [comment]
@Meowitsmadi Thanks for the quick changes. Everything looks good.

**Review by huss — APPROVED** [review]

**Pull request merged**